environment.sh

```diff
@@ -62,6 +66,11 @@ if [[ $create_env == '' ]]; then
   echo "Environment $env_name does not exist."
   create_env="yes"
   env_exists="no"
+elif [[ $make_new == 'yes' ]]; then
+  # User has requested to make a new environment
+  echo "User has requested to make a new environment."
```

Comment:
nit: seems weird to echo this to the user. Maybe just "Making a new environment."